AssertionError: Lists differ while running backend tests macOS with M1 chipset.

While running the backend tests on macOS with M1 chipset, the `AssertionError: List differ` error shows up. Here is the error log:

<img width="1440" alt="112790114-384f5600-907c-11eb-8e56-14d97347c0a6" src="https://user-images.githubusercontent.com/66058206/113092551-d3c30100-920b-11eb-94aa-d8c496db059b.png">
<img width="1440" alt="112790124-43a28180-907c-11eb-9d86-63ebd30ee610" src="https://user-images.githubusercontent.com/66058206/113092503-bd1caa00-920b-11eb-9614-1bf98d1e0e5b.png">
<img width="1440" alt="112790130-4604db80-907c-11eb-8a46-4d270d594a10" src="https://user-images.githubusercontent.com/66058206/113092505-be4dd700-920b-11eb-8d45-830249df6720.png">
<img width="1440" alt="112790136-48ffcc00-907c-11eb-9664-aa5cb6d8cb26" src="https://user-images.githubusercontent.com/66058206/113092508-bf7f0400-920b-11eb-8435-45e9d40494c1.png">

--- a/scripts/common_test.py
+++ b/scripts/common_test.py
@@ -918,7 +918,7 @@ class ManagedProcessTests(test_utils.TestBase):
             manager_should_have_sent_kill_signal: bool. Whether the manager
                 should have sent a kill signal to the process.
         """
-        proc_pattern = r'Process\((name=\'[a-z]+\', )?pid=%d\)' % (pid,)
+        proc_pattern = r'Process\((name=\'[A-Za-z]+\', )?pid=%d\)' % (pid,)
 
         expected_patterns = []
         if manager_should_have_sent_terminate_signal:

